tests/models/autoencoders/test_models_vae.py

```diff
@@ -102,8 +102,8 @@ def get_autoencoder_tiny_config(block_out_channels=None):
 
 
 def get_consistency_vae_config(block_out_channels=None, norm_num_groups=None):
-    block_out_channels = block_out_channels or [32, 64]
-    norm_num_groups = norm_num_groups or 32
+    block_out_channels = block_out_channels or [8, 16]
```

Comment:
Can we go smaller than 8 and 16? Is 2 and 4 possible?

Reply:
Further reduced in 2289d5a3.